clone_tests/clone_compilation_errors.py

```diff
@@ -49,6 +49,8 @@
         "Unknown keyword, only `weak` and `strong` are allowed"),
     ("clone!(v => move || {})",
         "You need to specify if this is a weak or a strong clone."),
+    ("clone!(@strong v => async move {println!(\"foo\");});",
+        "no rules expected the token `async`"),
```

Comment:
This seems like it could get a custom error message on our side. "async blocks are not supported by the clone! macro" or something like that.

Reply:
Fine by me!